partner_contact_personal_information_page/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+You can find a new page called *Personal Information* in the contact's form.
+
+For further information, please visit:
```

Comment:
From here to the end of the file is boilerplate. Remove it please.

Reply:
@Yajo Done